src/main/kotlin/org/lunakoly/quicklink/ui/Helpers.kt

```diff
@@ -37,13 +37,17 @@ const val FADE_OUT_DELAY_MS = 3000L
 
 fun Project.toast(message: String) {
     val statusBar = WindowManager.getInstance().getStatusBar(this)
+    val component = statusBar.component ?: let {
+        println("statusBar component is null")
+        return
+    }
```

Comment:
Usually, in such situations `run {}` is used, but personally I'd also be fine with `?: return println()`

Reply:
Sorry if I'm a bit tedious, otherwise we'd probably forget about this place if we postponed it to the next PR

Reply:
Will fix in this PR itself. I think then we can go with using `run`.  

`?: return println()` I dont really understand how this works, because we have already returned from the function.

Reply:
Pushed the changes with using `run`. Kept a message in println just in case if someone hits the null case and want to debug

Reply:
> `?: return println()` I dont really understand how this works, because we have already returned from the function.

 In Kotlin functions that don't return anything, in fact, return the `Unit` object. `return foo()` then works the same way as with any other function: it forwards `Unit`. You can view `return` is a shortcut for `return Unit`. But I see that it may be confusing (and it's not a recommended way to write Kotlin code anyways), so `run` is fine :)